nexus-repository-conan/src/main/java/org/sonatype/repository/conan/internal/hosted/ConanHostedHelper.java

```diff
@@ -0,0 +1,45 @@
+package org.sonatype.repository.conan.internal.hosted;
+
+import org.sonatype.nexus.repository.view.matchers.token.TokenMatcher;
+import org.sonatype.repository.conan.internal.AssetKind;
+import org.sonatype.repository.conan.internal.metadata.ConanCoords;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import static org.sonatype.repository.conan.internal.metadata.ConanMetadata.DIGEST;
+import static org.sonatype.repository.conan.internal.metadata.ConanMetadata.GROUP;
+import static org.sonatype.repository.conan.internal.metadata.ConanMetadata.PATH;
+import static org.sonatype.repository.conan.internal.metadata.ConanMetadata.PROJECT;
+import static org.sonatype.repository.conan.internal.metadata.ConanMetadata.STATE;
+import static org.sonatype.repository.conan.internal.metadata.ConanMetadata.VERSION;
+
+/**
+ * @since 1.0.0
+ */
+public class ConanHostedHelper
+{
+  public static final ObjectMapper MAPPER = new ObjectMapper();
+
+  public static final String CONAN_HOSTED_PREFIX = "conans/";
+
+  public static String getHostedAssetPath(final ConanCoords coord, final AssetKind assetKind) {
+    String path = String.format("%s/%s/%s/%s%s",
+        coord.getGroup(),
+        coord.getProject(),
+        coord.getVersion(),
+        coord.getChannel(),
+        coord.getSha() == null ? "" : "/packages/" + coord.getSha());
```

Comment:
StringUtils.EMPTY maybe?

Reply:
It was moved, maybe yes, but I do not want to change behavior

Reply:
I mean instead of ""

Reply:
I thought StringUtils.isEmpty )))

Reply:
fixed https://github.com/sonatype-nexus-community/nexus-repository-conan/pull/97/commits/35de5335f2bc22babed94ccec4989d164aa63098